phpstan.neon

```diff
@@ -0,0 +1,4 @@
+parameters:
+  level: 0
```

Comment:
I would suggest the following.

use: level: max here or lets say level 8 (9 is really hardcore)

and then create and include a baseline where we put all the issues into. Then we will shrink down the baseline issue by issue. That way new code is validated with full phpstan power.

Whole config would look like this:

```neon
includes:
    - phpstan-baseline.neon

parameters:
    level: 8
    
    paths:
        - src
       
```
You would generate the baseline with

`vendor/bin/phpstan analyse --generate-baseline=phpstan-baseline.neon`

and commit that